dandiapi/api/mail.py

```diff
@@ -43,8 +43,17 @@ def user_greeting_name(user: User, socialaccount: SocialAccount = None) -> str:
     return social_user['username']
 
 
-def build_message(subject: str, message: str, to: list[str], html_message: str | None = None):
-    email_message = mail.EmailMultiAlternatives(subject=subject, body=message, to=to)
+def build_message(  # noqa: PLR0913
+    to: list[str],
+    subject: str,
+    message: str,
+    html_message: str | None = None,
+    cc: list[str] | None = None,
+    bcc: list[str] | None = None,
+):
```

Comment:
I don't know if this PR is the place for it, but since you've altered this function already: I wonder if this function would be better off with keyword-only arguments. That would make the use of the function elsewhere much clearer at a glance.

Reply:
This method is honestly pretty superfluous, as it simply wraps the construction of a `mail.EmailMultiAlternatives` class instance. I think it'd be better to remove it entirely.

Reply:
I'll file an issue for this.